tests/unit/Utilities/010BasisFunctions_UnitTest.cpp

```diff
@@ -79,18 +85,17 @@ int main() {
 
             point1D[0] += -1e-8;
             ret = test->evalDeriv((point1D));
-            logger.assert_always((std::abs(ret[0] - 5.e7 * (x1 - x0)) < 1e-5),
-                                 "gradient");
-            logger.assert_always((std::abs(test->evalDeriv0((point1D)) -
-                                           5.e7 * (x1 - x0)) < 1e-5),
-                                 "derivative");
+            INFO("gradient");
+CHECK((std::abs(ret[0] - 5.e7 * (x1 - x0)) < 1e-5));
```

Comment:
This seems to be badly indented. Is that a requirement that the CHECK is at the start of the line?

Reply:
We can also require that we get the local catch2, but all the projects I have seen just put the catch header inside.

Reply:
@Laar the intendaton is fixed.